### PR TITLE
add alt text together during the 2021 diy a11y workshop

### PR DESCRIPTION
**Pull request by tonyfast**
this pull request collaboratively updates the retrolab image alt text.

## needs alt text

- [ ] line 92
- [ ] line 96
- [ ] line 100
- [ ] line 104
- [ ] line 110
- [ ] line 123
- [ ] line 127
- [ ] line 131
- [ ] line 135
- [ ] line 139
- [ ] line 145
- [ ] line 149

## learning resources
- An [alt text style guide proposed for Jupyter accessibility workshops](https://github.com/Quansight-Labs/jupyter-accessibility-workshops/blob/main/docs/alt-text-guide.md) that gives an overview on frequently asked questions (for transparency, I did help write this one). We've used this in some other projects too (like NumPy documentation). Most of the Jupyter-specific details can be ignored or replaced with other project-specific resources.
- A favorited resource, [W3's Alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).
- [WebAim's overview of alt text](https://webaim.org/techniques/alttext/).
- [Diagram Center's standards for alt text for graphs and charts](http://diagramcenter.org/table-of-contents-2.html#toc), which is one of the only resources in alt text I've encountered that covers these types of images.

Hope that helps!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,25 +89,25 @@ To access the main RetroLab tree (file browser):
 
 ### Files 📂 and Running Sessions 🏃‍♀️
 
-![tree](https://user-images.githubusercontent.com/591645/101952684-54c4a100-3bf9-11eb-8031-6900f6d3a445.gif)
+![tree ...](https://user-images.githubusercontent.com/591645/101952684-54c4a100-3bf9-11eb-8031-6900f6d3a445.gif)
```
Comment (tonyfast) [inline README.md:92]:
```suggestion
![a screenshot of jupyterlab retro's file browser](https://user-images.githubusercontent.com/591645/101952684-54c4a100-3bf9-11eb-8031-6900f6d3a445.gif)
```

Comment (JasonWeill) [inline README.md]:
```suggestion
![tree An animation of creating a new folder, moving an item into the folder, creating a new text file in the folder, and displaying the running notebooks and terminals in JupyterLab](https://user-images.githubusercontent.com/591645/101952684-54c4a100-3bf9-11eb-8031-6900f6d3a445.gif)
```

Comment (tonyfast) [inline README.md]:
```suggestion
![An animation showing the abilities of RetroLab. It shows the ability to make folders, move files, and create new ones.](https://user-images.githubusercontent.com/591645/101952684-54c4a100-3bf9-11eb-8031-6900f6d3a445.gif)
```

File: README.md [hunk 2]
```diff
@@ -89,25 +89,25 @@ To access the main RetroLab tree (file browser):
...
 
 ### Notebook 📒
 
-![notebook](https://user-images.githubusercontent.com/591645/101953039-efbd7b00-3bf9-11eb-9d34-3cb663a5ac43.gif)
+![notebook ...](https://user-images.githubusercontent.com/591645/101953039-efbd7b00-3bf9-11eb-9d34-3cb663a5ac43.gif)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![notebook An animation showing a user viewing a notebook, renaming a file, running cells, interacting with cell output, and modifying the Jupyter interface using the menu](https://user-images.githubusercontent.com/591645/101953039-efbd7b00-3bf9-11eb-9d34-3cb663a5ac43.gif)
```

File: README.md [hunk 3]
```diff
@@ -89,25 +89,25 @@ To access the main RetroLab tree (file browser):
...
 
 ### Open in a new Browser Tab by default
 
-![new-browser-tab](https://user-images.githubusercontent.com/591645/118811914-d9ed6980-b8ad-11eb-8362-f38627199f71.gif)
+![new-browser-tab ...](https://user-images.githubusercontent.com/591645/118811914-d9ed6980-b8ad-11eb-8362-f38627199f71.gif)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![new-browser-tab An animation of a user creating a notebook, opening it in a new browser tab and running a cell in it.](https://user-images.githubusercontent.com/591645/118811914-d9ed6980-b8ad-11eb-8362-f38627199f71.gif)
```

File: README.md [hunk 4]
```diff
@@ -89,25 +89,25 @@ To access the main RetroLab tree (file browser):
...
 
 ### Command Palette 🎨
 
-![command-palette](https://user-images.githubusercontent.com/591645/101953322-72ded100-3bfa-11eb-9b13-3a912e4f6844.gif)
+![command-palette ...](https://user-images.githubusercontent.com/591645/101953322-72ded100-3bfa-11eb-9b13-3a912e4f6844.gif)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![command-palette An animation of a user opening the command palette and selecting the "Restart Kernel and Clear All Outputs …" option](https://user-images.githubusercontent.com/591645/101953322-72ded100-3bfa-11eb-9b13-3a912e4f6844.gif)
```

File: README.md [hunk 5]
```diff
@@ -89,25 +89,25 @@ To access the main RetroLab tree (file browser):
...
 
 ### Themes 🌈
 
 Support for existing JupyterLab themes!
 
-![themes](https://user-images.githubusercontent.com/591645/101953333-75d9c180-3bfa-11eb-868f-af54d1ea7091.gif)
+![themes ...](https://user-images.githubusercontent.com/591645/101953333-75d9c180-3bfa-11eb-868f-af54d1ea7091.gif)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![themes An animation of a user changing the Jupyter interface to use the dark theme.](https://user-images.githubusercontent.com/591645/101953333-75d9c180-3bfa-11eb-868f-af54d1ea7091.gif)
```

File: README.md (hunk 3 above)
Comment (tonyfast) [inline README.md]:
```suggestion
![An example of creating a new notebook that opens a new tab and prompts to select a kernel. It shows executing a cell that shows and output then saves the result. It returns back to the file browser to show the notebooking in the running panel.](https://user-images.githubusercontent.com/591645/118811914-d9ed6980-b8ad-11eb-8362-f38627199f71.gif)
```

File: README.md [hunk 6]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
 
 Example with the [JupyterLab Miami Nights Theme](https://github.com/timkpaine/jupyterlab_miami_nights) installed as a prebuilt extension:
 
-![image](https://user-images.githubusercontent.com/591645/119634905-77e3b580-be13-11eb-9c4c-d187ebea9df8.png)
+![image ...](https://user-images.githubusercontent.com/591645/119634905-77e3b580-be13-11eb-9c4c-d187ebea9df8.png)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![image The Jupyter interface using the Miami Nights theme](https://user-images.githubusercontent.com/591645/119634905-77e3b580-be13-11eb-9c4c-d187ebea9df8.png)
```

File: README.md [hunk 7]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 ### Zen Mode 😌
 
-![compact-zen-mode](https://user-images.githubusercontent.com/591645/101923740-149cf880-3bd0-11eb-9617-e3349a76d034.gif)
+![compact-zen-mode ...](https://user-images.githubusercontent.com/591645/101923740-149cf880-3bd0-11eb-9617-e3349a76d034.gif)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![compact-zen-mode An animation of a user using the command palette to enable Zen Mode in JupyterLab](https://user-images.githubusercontent.com/591645/101923740-149cf880-3bd0-11eb-9617-e3349a76d034.gif)
```

File: README.md [hunk 8]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 ### Terminals 🖥️
 
-![terminals](https://user-images.githubusercontent.com/591645/118793525-fc28bc80-b898-11eb-9e02-d5a80ad8ddb8.gif)
+![terminals ...](https://user-images.githubusercontent.com/591645/118793525-fc28bc80-b898-11eb-9e02-d5a80ad8ddb8.gif)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![terminals An animation of a user launching a terminal in a new browser tab and executing a command in it](https://user-images.githubusercontent.com/591645/118793525-fc28bc80-b898-11eb-9e02-d5a80ad8ddb8.gif)
```

File: README.md [hunk 9]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 ### File Editor 🖊️
 
-![image](https://user-images.githubusercontent.com/591645/101953590-e2ed5700-3bfa-11eb-9fee-0b6d964f0949.png)
+![image ...](https://user-images.githubusercontent.com/591645/101953590-e2ed5700-3bfa-11eb-9fee-0b6d964f0949.png)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![image A RetroLab editor of a Python file with a function and a statement in it](https://user-images.githubusercontent.com/591645/101953590-e2ed5700-3bfa-11eb-9fee-0b6d964f0949.png)
```

File: README.md [hunk 10]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 ### Compact View on Mobile Devices 📱
 
-![mobile](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
+![mobile ...](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
```
Comment (JasonWeill) [inline README.md]:
```suggestion
![mobile Animation of a user on a mobile phone-sized screen using a compact touch interface in JupyterLab](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
```

File: README.md [hunk 11]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 Install new extensions easily!
 
-![federated-extensions](https://user-images.githubusercontent.com/591645/118813222-38671780-b8af-11eb-835b-f2865bab5c62.gif)
+![federated-extensions ...](https://user-images.githubusercontent.com/591645/118813222-38671780-b8af-11eb-835b-f2865bab5c62.gif)
 
 ### Switch between JupyterLab and RetroLab easily ↔️
 
-![jupyterlab-switch](https://user-images.githubusercontent.com/591645/118816963-05bf1e00-b8b3-11eb-92a2-899288471011.gif)
+![jupyterlab-switch ...](https://user-images.githubusercontent.com/591645/118816963-05bf1e00-b8b3-11eb-92a2-899288471011.gif)
```
Comment (isabela-pf) [inline README.md]: Writing this!

```suggestion
![An animation of opening RetroLab in another tab from a button in the JupyterLab toolbar.](https://user-images.githubusercontent.com/591645/118816963-05bf1e00-b8b3-11eb-92a2-899288471011.gif)
```

File: README.md [hunk 12]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
-![mobile](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
+![mobile ...](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
 
 ### Support for prebuilt extensions 🧩
 
 Install new extensions easily!
 
-![federated-extensions](https://user-images.githubusercontent.com/591645/118813222-38671780-b8af-11eb-835b-f2865bab5c62.gif)
+![federated-extensions ...](https://user-images.githubusercontent.com/591645/118813222-38671780-b8af-11eb-835b-f2865bab5c62.gif)
```
Comment (isabela-pf) [inline README.md]: Working on this!

```suggestion
![An animation of installing an extension directly in a RetroLab by running commands in a notebook cell.](https://user-images.githubusercontent.com/591645/118813222-38671780-b8af-11eb-835b-f2865bab5c62.gif)
```

Comment (JasonWeill) [inline README.md]:
```suggestion
![federated-extensions Animation of a user running `pip install` in a JupyterLab notebook to install the toolbar extension](https://user-images.githubusercontent.com/591645/118813222-38671780-b8af-11eb-835b-f2865bab5c62.gif)
```

File: README.md (hunk 11 above)
Comment (JasonWeill) [inline README.md]:
```suggestion
![jupyterlab-switch Animation of a user opening RetroLab from within JupyterLab](https://user-images.githubusercontent.com/591645/118816963-05bf1e00-b8b3-11eb-92a2-899288471011.gif)
```

File: README.md (hunk 10 above)
Comment (isabela-pf) [inline README.md]: Working on this!

```suggestion
![An animation of scrolling through a notebook and the compact toolbar in RetroLab on mobile.](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
```

**Comment by isabela-pf** [comment]
If anyone wants/needs help reviewing or cleaning up suggestions, let me know! I've done this a few times and am happy to help.

**Comment by tonyfast** [comment]
i'd love some help with review. i won't be able to get to review until tomorrow afternoon. any :open_hands: would be appreciated.

**Comment by isabela-pf** [comment]
I'm going to commit a mix of suggestions so that everyone who made a suggestion is included in the commit history, and then edit it locally. A lot of these with multiple suggestions have things I'd like to combine, but since I can't commit both I figured this would be the best middle ground to credit people's efforts.

File: README.md [hunk 13]
```diff
@@ -89,25 +89,25 @@ To access the main RetroLab tree (file browser):
 
 ### Files 📂 and Running Sessions 🏃‍♀️
 
-![tree](https://user-images.githubusercontent.com/591645/101952684-54c4a100-3bf9-11eb-8031-6900f6d3a445.gif)
+![An animation showing the abilities of RetroLab. It shows the ability to make folders, move files, and create new ones.](https://user-images.githubusercontent.com/591645/101952684-54c4a100-3bf9-11eb-8031-6900f6d3a445.gif)
 
 ### Notebook 📒
 
-![notebook](https://user-images.githubusercontent.com/591645/101953039-efbd7b00-3bf9-11eb-9d34-3cb663a5ac43.gif)
+![notebook An animation showing a user viewing a notebook, renaming a file, running cells, interacting with cell output, and modifying the Jupyter interface using the menu](https://user-images.githubusercontent.com/591645/101953039-efbd7b00-3bf9-11eb-9d34-3cb663a5ac43.gif)
```
Comment (tonyfast) [inline README.md]:
```suggestion
![An animation showing a user viewing a notebook, renaming a file, running cells, interacting with cell output, and modifying the Jupyter interface using the menu](https://user-images.githubusercontent.com/591645/101953039-efbd7b00-3bf9-11eb-9d34-3cb663a5ac43.gif)
```

File: README.md [hunk 14]
```diff
@@ -89,25 +89,25 @@ To access the main RetroLab tree (file browser):
...
 
 ### Open in a new Browser Tab by default
 
-![new-browser-tab](https://user-images.githubusercontent.com/591645/118811914-d9ed6980-b8ad-11eb-8362-f38627199f71.gif)
+![An example of creating a new notebook that opens a new tab and prompts to select a kernel. It shows executing a cell that shows and output then saves the result. It returns back to the file browser to show the notebooking in the running panel.](https://user-images.githubusercontent.com/591645/118811914-d9ed6980-b8ad-11eb-8362-f38627199f71.gif)
 
 ### Command Palette 🎨
 
-![command-palette](https://user-images.githubusercontent.com/591645/101953322-72ded100-3bfa-11eb-9b13-3a912e4f6844.gif)
+![command-palette An animation of a user opening the command palette and selecting the "Restart Kernel and Clear All Outputs …" option](https://user-images.githubusercontent.com/591645/101953322-72ded100-3bfa-11eb-9b13-3a912e4f6844.gif)
```
Comment (tonyfast) [inline README.md]:
```suggestion
![An animation of a user opening the command palette and selecting the "Restart Kernel and Clear All Outputs …" option](https://user-images.githubusercontent.com/591645/101953322-72ded100-3bfa-11eb-9b13-3a912e4f6844.gif)
```

File: README.md [hunk 15]
```diff
@@ -89,25 +89,25 @@ To access the main RetroLab tree (file browser):
...
 
 ### Themes 🌈
 
 Support for existing JupyterLab themes!
 
-![themes](https://user-images.githubusercontent.com/591645/101953333-75d9c180-3bfa-11eb-868f-af54d1ea7091.gif)
+![themes An animation of a user changing the Jupyter interface to use the dark theme.](https://user-images.githubusercontent.com/591645/101953333-75d9c180-3bfa-11eb-868f-af54d1ea7091.gif)
```
Comment (tonyfast) [inline README.md]:
```suggestion
![An animation of a user changing the Jupyter interface to use the dark theme.](https://user-images.githubusercontent.com/591645/101953333-75d9c180-3bfa-11eb-868f-af54d1ea7091.gif)
```

File: README.md [hunk 16]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
 
 Example with the [JupyterLab Miami Nights Theme](https://github.com/timkpaine/jupyterlab_miami_nights) installed as a prebuilt extension:
 
-![image](https://user-images.githubusercontent.com/591645/119634905-77e3b580-be13-11eb-9c4c-d187ebea9df8.png)
+![image The Jupyter interface using the Miami Nights theme](https://user-images.githubusercontent.com/591645/119634905-77e3b580-be13-11eb-9c4c-d187ebea9df8.png)
```
Comment (tonyfast) [inline README.md]:
```suggestion
![The Jupyter interface using the Miami Nights theme](https://user-images.githubusercontent.com/591645/119634905-77e3b580-be13-11eb-9c4c-d187ebea9df8.png)
```

File: README.md [hunk 17]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 ### Zen Mode 😌
 
-![compact-zen-mode](https://user-images.githubusercontent.com/591645/101923740-149cf880-3bd0-11eb-9617-e3349a76d034.gif)
+![compact-zen-mode An animation of a user using the command palette to enable Zen Mode in JupyterLab](https://user-images.githubusercontent.com/591645/101923740-149cf880-3bd0-11eb-9617-e3349a76d034.gif)
```
Comment (tonyfast) [inline README.md]:
```suggestion
![An animation of a user using the command palette to enable Zen Mode in JupyterLab](https://user-images.githubusercontent.com/591645/101923740-149cf880-3bd0-11eb-9617-e3349a76d034.gif)
```

File: README.md [hunk 18]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 ### Terminals 🖥️
 
-![terminals](https://user-images.githubusercontent.com/591645/118793525-fc28bc80-b898-11eb-9e02-d5a80ad8ddb8.gif)
+![terminals An animation of a user launching a terminal in a new browser tab and executing a command in it](https://user-images.githubusercontent.com/591645/118793525-fc28bc80-b898-11eb-9e02-d5a80ad8ddb8.gif)
```
Comment (tonyfast) [inline README.md]:
```suggestion
![An animation of a user launching a terminal in a new browser tab and executing a command in it](https://user-images.githubusercontent.com/591645/118793525-fc28bc80-b898-11eb-9e02-d5a80ad8ddb8.gif)
```

File: README.md [hunk 19]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 ### File Editor 🖊️
 
-![image](https://user-images.githubusercontent.com/591645/101953590-e2ed5700-3bfa-11eb-9fee-0b6d964f0949.png)
+![image A RetroLab editor of a Python file with a function and a statement in it](https://user-images.githubusercontent.com/591645/101953590-e2ed5700-3bfa-11eb-9fee-0b6d964f0949.png)
```
Comment (tonyfast) [inline README.md]:
```suggestion
![A RetroLab editor of a Python file with a function and a statement in it](https://user-images.githubusercontent.com/591645/101953590-e2ed5700-3bfa-11eb-9fee-0b6d964f0949.png)
```

File: README.md [hunk 20]
```diff
@@ -120,33 +120,33 @@ jupyter retro --retro-logo
...
 
 ### Compact View on Mobile Devices 📱
 
-![mobile](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
+![mobile Animation of a user on a mobile phone-sized screen using a compact touch interface in JupyterLab](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
```
Comment (tonyfast) [inline README.md]:
```suggestion
![Animation of a user on a mobile phone-sized screen using a compact touch interface in JupyterLab](https://user-images.githubusercontent.com/591645/101995448-2793f380-3cca-11eb-8971-067dd068ccbe.gif)
```

**Pull request merged**